### ISSUE TITLE
Create a GUI for the video export module

### ISSUE THREAD

**Issue by jgcodes2020** [musescore/MuseScore#25793]
### Your idea

MuseScore already has the necessary functionality to generate videos (see `MUE_BUILD_VIDEOEXPORT_MODULE`). Provide a GUI to configure and export video locally through the code in this module.

### Problem to be solved

As of writing this issue, the only officially-supported way to generate video is by uploading it to `musescore.com` and using "Send to YouTube". It would be a lot less painful if I could simply generate the video locally.

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by cbjeukendrup** [comment]
There is slightly more to it than just a UI though: the video generation feature depends on FFmpeg. Distributing a copy of FFmpeg packaged with MuseScore Studio is not allowed (or rather, requires Muse to pay a lot to FFmpeg, I believe). So we'll need to tell users to install FFmpeg from some other source, and then work with that external FFmpeg installation. So far so good, but the trouble is that every OS's package manager would install a different version of FFmpeg, and as the Audacity team told us, different FFmpeg versions are not exactly nicely compatible. 

So that is why this is slightly less trivial than just building a UI, and thus why it hasn't been done yet.

**Comment by rmattes** [comment]
Yes, indeed - this can't be a license problem. Both MuseScore and FFmpeg are GPL-licensed. I have a nagging feeling that such a feature would interfere with musescore.com's business interests and that this is the reason for it not being exposed to the public.

**Comment by cbjeukendrup** [comment]
Apparently it was not license problems but patent problems. This is what FFmpeg say about it themselves: https://www.ffmpeg.org/legal.html
So that's quite vague and doesn't state very clearly whether we would be able to use it or not, being an open source project. The impression that I get from it, is that as soon as you enable the _useful_ parts of FFmpeg, i.e. H.264 and MPEG-4, people will come after you for money, but to be fair, that's also not _really_ what they are saying. 

Anyway, the only thing that I know for sure is that there must be a reason that Audacity is (and has been before the acquisition by Muse) going through the trouble of working with external FFmpeg.
